### PR TITLE
Feature/textview

### PR DESCRIPTION
**Pull request by hbons**
Done!

### CODE REVIEW

File: FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ protected override string OnGenerate ()
 
 	public abstract class FigmaClassBase
 	{
-		 protected const string InitializeComponentMethodName = "InitializeComponent";
+		 protected const string InitializeComponentMethodName = "Initialize";
```
Comment (netonjm) [inline FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs:92]: ???

**Comment by netonjm** [comment]
rebase this in top of master

File: FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs (hunk 1 above)
Comment (hbons) [reply FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs]: sorry, rebase on master went wrong. try again. :)

File: FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/Converters/TextViewConverter.cs [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+			return view;
+		}
+
+		protected override StringBuilder OnConvertToCode(FigmaCodeNode currentNode, FigmaCodeNode parentNode, FigmaCodeRendererService rendererService)
+		{
+			var instance = (FigmaFrameEntity)currentNode.Node;
+
+			string textViewName = currentNode.Name;
+
+			var name = currentNode.Name + "ScrollView";
```
Comment (netonjm) [inline FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/Converters/TextViewConverter.cs:97]: why this?

Comment (hbons) [reply FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/Converters/TextViewConverter.cs:97]: This is like in Xcode. TextViews are always wrapped in a ScrollView or they don't work and we need to generate different names. Here's an example of the code generated: https://gist.github.com/hbons/388004fd9abec8877fc4cff9abed5382

Comment (netonjm) [reply FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/Converters/TextViewConverter.cs:97]: Ahh ! I see this is for the enclosed scrollview 👍

**Pull request merged**